### DISCUSSION TITLE
How to check Cueobserve liveness

### DISCUSSION THREAD

**Discussion by ArunMu21** [cuebook/CueObserve#231] [Q&A]
Hello Team,

We are using cueobserve release 0.2 version where all cueobserve components are in one docker image only. We have deployed this cueobserve image into Kubernetes. it is working however after every 6-7 days it automatically stopped scheduling the task at back-end(Like no anomaly definition got triggered at schedule time). Once we restart the container everything works normally.

Due to this unexpected failure of cueobserve we have manually check the status of cueobserve daily so we want to apply some cueobserve liveness check in Kubernetes deployment file only.

We tried below liveness check in our deployment file but it does not seems to work:

`      livenessProbe:
          exec:
            command:
            - bash
            - -c
            - celery -A app inspect ping -d celery@$HOSTNAME`

@sachinkbansal  Can you please help us on how can we can track the liveness of all the components of cueobserve

**Comment by ArunMu21** [comment]
Hello Team,

Can you please help us here?

**Comment by vincue** [comment]
This is probably due to a bug which was resolved in PR #134. 
Please consider upgrading to `release 0.3.2`. It is backward compatible and supports kubernetes deployment using helm. Many of the bugs from earlier versions are resolved here.
[Latest Release (0.3.2)](https://github.com/cuebook/cueobserve)
